src/pages/project/project_wizard/ProjectDetailsForm.tsx

```diff
@@ -1,10 +1,95 @@
+import {
+  CristinProject,
+  ProjectFieldName,
+  ProjectOrganization,
+  SaveCristinProject,
+  TypedLabel,
+} from '../../../types/project.types';
+import { dataTestId } from '../../../utils/dataTestIds';
+import { getLanguageString } from '../../../utils/translation-helpers';
+import { OrganizationSearchField } from '../../basic_data/app_admin/OrganizationSearchField';
+import { isRekProject } from '../../registration/description_tab/projects_field/projectHelpers';
+import { ProjectFundingsField } from './ProjectFundingsField';
+import { FormBox } from './styles';
+
+interface ProjectDescriptionFormProps {
+  project: CristinProject;
+}
+
+export const ProjectDetailsForm = ({ project }: ProjectDescriptionFormProps) => {
+  const { t } = useTranslation();
+  const { values, setFieldValue, setFieldTouched, touched, errors } = useFormikContext<SaveCristinProject>();
+  const thisIsRekProject = isRekProject(project);
 
-export const ProjectDetailsForm = () => {
   return (
     <ErrorBoundary>
-      <Box sx={{ bgcolor: 'secondary.light', padding: '1.75rem 1.25rem' }}>test 2</Box>
+      <Box sx={{ display: 'flex', flexDirection: 'column', gap: '1rem' }}>
+        <FormBox>
+          <Typography variant="h2">{t('project.coordinating_institution')}</Typography>
+          <Field name={ProjectFieldName.CoordinatingInstitution}>
+            {({ field }: FieldProps<ProjectOrganization>) => (
+              <OrganizationSearchField
+                disabled={thisIsRekProject}
+                label={t('project.coordinating_institution')}
+                onChange={(selectedInstitution) => {
+                  const selectedCoordinatingInstitution: ProjectOrganization = {
+                    type: 'Organization',
+                    id: selectedInstitution?.id ?? '',
+                    labels: selectedInstitution?.labels ?? {},
+                  };
+                  setFieldValue(field.name, selectedCoordinatingInstitution);
+                }}
+                errorMessage={
+                  touched.coordinatingInstitution?.id && !!errors.coordinatingInstitution?.id
+                    ? errors.coordinatingInstitution?.id
+                    : undefined
+                }
+                fieldInputProps={{
+                  ...field,
+                  onBlur: () => {
+                    setFieldTouched(`${field.name}.id`);
+                  },
+                }}
+                selectedValue={field.value}
+                customDataTestId={dataTestId.registrationWizard.description.projectForm.coordinatingInstitutionField}
+              />
+            )}
+          </Field>
+        </FormBox>
+        <FormBox>
+          <Typography variant="h2" sx={{ pb: '0.25rem' }}>
+            {t('project.form.category')}
+          </Typography>
+          <Field name={ProjectFieldName.Categories}>
+            {({ field, form: { setFieldValue } }: FieldProps<TypedLabel[]>) => (
```

Comment:
allerede importert på linje 26
```suggestion
            {({ field }: FieldProps<TypedLabel[]>) => (
```